cmd/fyne/internal/commands/flags.go

```diff
@@ -36,3 +38,191 @@ func (k *keyValueFlag) String() string {
+			Destination: dst,
+		}
+	},
+	"app-version": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "app-version",
+			Usage:       "set version number in the form x, x.y or x.y.z semantic version",
+			Destination: dst,
+		}
+	},
+	"category": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "category",
+			Usage:       "macos: category of the app for store listing",
+			Destination: dst,
+		}
+	},
+	"certificate": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "certificate",
+			Aliases:     []string{"cert"},
+			Usage:       "ios/macos/windows: name of the certificate to sign the build",
+			Destination: dst,
+		}
+	},
+	"developer": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "developer",
+			Aliases:     []string{"dev"},
+			Usage:       "windows: the developer identity for your Microsoft store account",
+			Destination: dst,
+		}
+	},
+	"executable": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "executable",
+			Aliases:     []string{"exe"},
+			Usage:       "set path to executable",
+			DefaultText: "current directory main binary",
+			Destination: dst,
+		}
+	},
+	"icon": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "icon",
+			Usage:       "set name of the application icon file",
+			Destination: dst,
+		}
+	},
+	"install-dir": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "install-dir",
+			Aliases:     []string{"o"},
+			Usage:       "specify location to install to, rather than the OS default",
+			Destination: dst,
+		}
+	},
+	"key-name": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "key-name",
+			Usage:       "android: alias for the signer's private key, which is needed when reading a .keystore file",
+			Destination: dst,
+		}
+	},
+	"key-pass": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "key-pass",
+			Usage:       "android: password for the signer's private key, which is needed if the private key is password-protected",
+			DefaultText: "read from stdin",
+			Destination: dst,
+		}
+	},
+	"key-store": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "key-store",
+			Usage:       "android: location of .keystore file containing signing information",
+			Destination: dst,
+		}
+	},
+	"key-store-pass": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "key-store-pass",
+			Usage:       "android: password for the .keystore file",
+			DefaultText: "read from stdin",
+			Destination: dst,
+		}
+	},
+	"name": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "name",
+			Usage:       "set name of the application",
+			DefaultText: "executable file name",
+			Destination: dst,
+		}
+	},
+	"output": func(dst *string) cli.Flag {
+		return &cli.StringFlag{
+			Name:        "output",
```

Comment:
See comment above. This might want a "-o" alias?

Reply:
I removed the "o" alias from the "install-dir" flag now and moved it here.